test/es-module/test-esm-json.mjs

```diff
@@ -1,7 +1,29 @@
 // Flags: --experimental-json-modules
 import '../common/index.mjs';
-import { strictEqual } from 'assert';
+import { path } from '../common/fixtures.mjs';
+import { strictEqual, ok } from 'assert';
+import { spawn } from 'child_process';
 
 import secret from '../fixtures/experimental.json';
 
 strictEqual(secret.ofLife, 42);
+
+// Test warning message
+const child = spawn(process.execPath, [
+  '--experimental-json-modules',
+  path('/es-modules/json-modules.mjs')
+]);
+
+let stderr = '';
+child.stderr.setEncoding('utf8');
+child.stderr.on('data', (data) => {
+  stderr += data;
+});
+child.on('close', (code, signal) => {
+  strictEqual(code, 0);
+  strictEqual(signal, null);
+  ok(stderr.toString().includes(
+    'ExperimentalWarning: JSON Module is an experimental feature. ' +
```

Comment:
Similarly, I think we should make this _**Importing JSON modules** is an experimental feature_.

Reply:
Updated